BENCHMARKS.md

```diff
@@ -5,51 +5,51 @@ Performance: native vs wrapped
 
 benchmark | subject | revs | mem_peak | time_rev | diff
  --- | --- | --- | --- | --- | --- 
-ArrayValueBench | arrayDiffSameFunction | 10 | 2,600,176b | 0.35ms | 1.10x
-ArrayValueBench | arrayDiffSameWrapped | 10 | 3,781,632b | 0.69ms | 2.15x
-ArrayValueBench | arrayDiffSubsetFunction | 10 | 2,575,600b | 0.32ms | 1.00x
-ArrayValueBench | arrayDiffSubsetWrapped | 10 | 3,757,112b | 0.86ms | 2.66x
-ArrayValueBench | arrayDiffComparatorFunction | 10 | 2,895,128b | 17.02ms | 52.83x
-ArrayValueBench | arrayDiffComparatorWrapped | 10 | 3,757,120b | 17.40ms | 54.03x
+ArrayValueBench | arrayDiffSameFunction | 1000 | 3,306,376b | 0.21ms | 1.10x
+ArrayValueBench | arrayDiffSameWrapped | 1000 | 3,468,240b | 0.21ms | 1.11x
+ArrayValueBench | arrayDiffSubsetFunction | 1000 | 3,220,360b | 0.19ms | 1.00x
+ArrayValueBench | arrayDiffSubsetWrapped | 1000 | 3,869,672b | 0.25ms | 1.32x
+ArrayValueBench | arrayDiffComparatorFunction | 1000 | 3,535,792b | 103.11ms | 542.65x
+ArrayValueBench | arrayDiffComparatorWrapped | 1000 | 3,870,264b | 102.03ms | 536.98x
```

Comment:
That's a big performance degradation :O

Reply:
Yes, it's strange but it's the same with plain PHP.

This benchmark is on PHP8, different machine then previous, still it seems like degradation of `array_udiff` or performance improvement of `array_diff` :thinking:

Reply:
I've fixed the benchmarks without xdebug :boar: